### PR TITLE
Change the 'mul_op' to 'matmul_op' in fc.

### LINKED ISSUE #8100: Change the 'mul_op' in fc to 'matmul_op'.

### PR DESCRIPTION
**Pull request by peterzhang2029**
resolve https://github.com/PaddlePaddle/Paddle/issues/8100

### CODE REVIEW

**Comment by lcy-seso** [comment]
为什么需要这个修改，能否在issue里面说明一下？相关的issue没有描述。

File: paddle/operators/matmul_op.cc [hunk 1]
```diff
@@ -162,6 +200,37 @@ class MatMulOpMaker : public framework::OpProtoAndCheckerMaker {
...
+              matrix (the width of the matrix). As a result, height of the
+              flattened matrix is equal to the product of $X$'s first
+              `x_num_col_dims` dimensions' sizes, and width of the flattened
+              matrix is equal to the product of $X$'s last `rank(x) - num_col_dims`
+              dimensions' size. For example, suppose $X$ is a 6-dimensional
+              tensor with the shape [2, 3, 4, 5, 6], and `x_num_col_dims` = 3.
+              Thus, the flattened matrix will have a shape [2 x 3 x 4, 5 x 6] =
+              [24, 30]. The default value 0 indicates the input is a 2-D Matrix.
+        )DOC")
+        .SetDefault(0)
```
Comment (dzhwinter) [inline paddle/operators/matmul_op.cc]: double check the default value, Please.

**Comment by lcy-seso** [comment]
mul_op 和 matmul_op 是两种计算。这个PR如果是为了将两者统一，最好再仔细确认一下计算逻辑。这两个op不能简单的删掉一个。

**Comment by luotao1** [comment]
感谢您给PaddlePaddle贡献代码。由于Paddle V1/V2版本已不再维护，相关代码也已从develop分支上删除，因此关闭您的PR，欢迎您向Paddle最新版-Fluid贡献代码。
Thanks for contributing to PaddlePaddle! Since V1/V2 will not be maintained anymore, and related codes have been deleted from develop branch as well, we close this PR. Welcome to contribute to Fluid——the latest version of PaddlePaddle.

**Pull request closed without merging**